features/step_definitions/user_steps.rb

```diff
@@ -136,16 +140,6 @@ def page_heading_text(page)
   click_on('Register')
 end
 
-Given('they go back to the country picker') do
-  visit(URI.join(env('frontend'), 'choose-a-country'))
-  assert_text('Use a digital identity from another European country')
-end
-
-Given('they go back to the journey picker page') do
-  visit(URI.join(env('frontend'), 'prove-identity'))
-  assert_text('Prove your identity to continue')
-end
-
```

Comment:
This is fine but can we rename the `they choose to go back to the {string} page` method to `they go back to the {string} page` then? The `choose to` doesn't make sense - they don't select from a list or anything, they simply press the back button.

Reply:
Point noted and changes made accordingly. Can you please review again. Thanks